syntax = "proto3";

message JsonResponse{
	string response = 1;
}

service DataServer{
	rpc PostResponse (JsonResponse) returns (JsonResponse) {}
}
